libs/judicial-system/types/src/lib/user.ts

```diff
@@ -60,7 +60,11 @@ export const prosecutionRoles: string[] = [
   UserRole.REPRESENTATIVE,
 ]
 
-export function isProsecutionRole(role: string): boolean {
+export function isProsecutionRole(role?: string): boolean {
+  if (!role) {
```

Comment:
Just wondering, why would we call this with an empty string? Wouldn't we then already know that no role is not a prosecution role?

Reply:
You are correct, you wouldn't call this with an empty string. However, the way our api is set up, this fields comes to the frontend as potentially undefined. So instead of checking everywhere before we use this function if role is not undefined, I decided to do it just once, inside the function.